lib/trailblazer/rails/test/integration.rb

```diff
@@ -1,3 +1,4 @@
+gem 'minitest-rails-capybara'
```

Comment:
can you move this into the `Gemfile/.gemspec` please?

Reply:
since we dont depend on it. we need to show which gem has that file instead of failing at the require.